### ISSUE TITLE
Does not run on Ubuntu 14.04 trusty

### ISSUE THREAD

**Issue by probonopd** [kiwix/kiwix-build#258]
Thank you very much for providing an AppImage for Linux, highly appreciated. However, it currently does not run on Ubuntu 14.04 trusty (which is a requirement to be included in the AppImageHub central directory of available AppImages):

```
AppRun: /lib/x86_64-linux-gnu/libz.so.1: version `ZLIB_1.2.9' not found (required by /run/firejail/appimage/.appimage-7188/usr/bin/../lib/libpng16.so.16)
/run/firejail/appimage/.appimage-7188/AppRun: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by /run/firejail/appimage/.appimage-7188/usr/bin/../lib/libsystemd.so.0)
/run/firejail/appimage/.appimage-7188/AppRun: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.22' not found (required by /run/firejail/appimage/.appimage-7188/usr/bin/../lib/libgnutls.so.30)
/run/firejail/appimage/.appimage-7188/AppRun: /lib/x86_64-linux-gnu/libgpg-error.so.0: no version information available (required by /run/firejail/appimage/.appimage-7188/usr/bin/../lib/libgcrypt.so.20)
/run/firejail/appimage/.appimage-7188/AppRun: /lib/x86_64-linux-gnu/libkeyutils.so.1: version `KEYUTILS_1.5' not found (required by /run/firejail/appimage/.appimage-7188/usr/bin/../lib/libkrb5.so.3)
```

suggests that this was compiled on a system newer than the oldest still-supported distribution LTS release, which currently is Ubuntu 14.04. Can you please compile it there, so that the resulting AppImage will run on all still-supported versions of Ubuntu (and other Linux distributions from the same age range)? https://launchpad.net/~beineri contains recent versions of Qt for trusty, those are known to work well.

References:
* https://github.com/AppImage/appimage.github.io/pull/621#issuecomment-423878661
* https://github.com/AppImage/AppImageKit/wiki/Creating-AppImages#creating-appimages-that-are-compatible-with-many-systems

**Comment by kelson42** [comment]
Yes, works only with recent libc, Ubuntu 18.04. For multiple reasons, the amount of work to get it compiling on 14.04, is too high. We might find a solution to get it work on 16.04, but this is still quite open. Sorry/

**Comment by probonopd** [comment]
> For multiple reasons, the amount of work to get it compiling on 14.04, is too high

Can you give me some hints please as to why this is?

Especially offline PCs tend to stay on old versions of OSes for a _very_ long time (I occasionally have to support a Ubuntu gutsy box that is still running an OS from 11 years ago).

**Comment by mgautierfr** [comment]
> Can you give me some hints please as to why this is?

Package `qtwebengine5-dev` is only available for ubuntu bionic (18.04LTS).

**Comment by probonopd** [comment]
```
libc.so.6: version `GLIBC_2.25' not found (required by /run/firejail/appimage/.appimage-8344/usr/bin/../lib/libsystemd.so.0)
```

Glibc 2.25 is from 2017, so it's not only too new for trusty but it's still too new for xenial!

Can you please build this on xenial?
